README.md

```diff
@@ -18,6 +18,14 @@ CLI to open PEPs in your browser.
 python3 -m pip install --upgrade pepotron
 ```
 
+### With [pipx][pipx]
+
+```bash
+pipx install pepotron
+```
+
+[pipx]: https://github.com/pypa/pipx
```

Comment:
Not sure if you have a preference on Markdown linking syntax, but I'm happy to change this to just be inline:
```markdown
### With [pipx](https://github.com/pypa/pipx)
```

I assume that syntax approach that I have in there now renders out fine on PyPI, given that building a wheel and sdist with `python -m build` and then checking with `twine check` passed.

Reply:
I normally do them inline (in body text), but this way looks cleaner for a title. Thanks!